assets/javascripts/kitten/components/notifications/alert.js

```diff
@@ -16,6 +18,7 @@ export class Alert extends React.Component {
   handleCloseClick() {
     this.setState({
       show: false,
+      height: domElementHelper.getComputedHeight(this.container),
```

Comment:
On pourrait peut-être ajouter un petit commentaire. Genre `// The css animation on the close button requires a fixed height`.